packages/sanity/src/core/releases/tool/components/ReleasePublishAllButton/ReleaseScheduleButton.tsx

```diff
@@ -0,0 +1,197 @@
+import {Flex, Stack, Text, useToast} from '@sanity/ui'
+import {useCallback, useMemo, useState} from 'react'
+
+import {Button, Dialog} from '../../../../../ui-components'
+import {MONTH_PICKER_VARIANT} from '../../../../../ui-components/inputs/DateInputs/calendar/Calendar'
+import {type CalendarLabels} from '../../../../../ui-components/inputs/DateInputs/calendar/types'
+import {DateTimeInput} from '../../../../../ui-components/inputs/DateInputs/DateTimeInput'
+import {getCalendarLabels} from '../../../../form/inputs/DateInputs/utils'
+import {useDateTimeFormat} from '../../../../hooks'
+import {Translate, useTranslation} from '../../../../i18n'
+import {type ReleaseDocument} from '../../../../store'
+import {useReleaseOperations} from '../../../../store/release/useReleaseOperations'
+import {ScheduledRelease} from '../../../__telemetry__/releases.telemetry'
+import {releasesLocaleNamespace} from '../../../i18n'
+import {type DocumentInRelease} from '../../detail/useBundleDocuments'
+
+interface ReleaseScheduleButtonProps {
+  release: ReleaseDocument
+  documents: DocumentInRelease[]
+  disabled?: boolean
+}
+
+export const ReleaseScheduleButton = ({
+  release,
+  disabled,
+  documents,
+}: ReleaseScheduleButtonProps) => {
+  const toast = useToast()
+  const {schedule} = useReleaseOperations()
+  const {t} = useTranslation(releasesLocaleNamespace)
+  const telemetry = useTelemetry()
+  const [status, setStatus] = useState<'idle' | 'confirm' | 'scheduling'>('idle')
+
+  const isValidatingDocuments = documents.some(({validation}) => validation.isValidating)
+  const hasDocumentValidationErrors = documents.some(({validation}) => validation.hasError)
+  const isScheduleButtonDisabled = disabled || isValidatingDocuments || hasDocumentValidationErrors
+
+  const handleConfirmSchedule = useCallback(async () => {
+    try {
+      setStatus('scheduling')
+      await schedule(release._id, new Date(release.metadata.intendedPublishAt!))
+      telemetry.log(ScheduledRelease)
+      toast.push({
+        closable: true,
+        status: 'success',
+        title: (
+          <Text muted size={1}>
+            <Translate
+              t={t}
+              i18nKey="toast.schedule.success"
+              values={{title: release.metadata.title}}
+            />
+          </Text>
+        ),
+      })
+    } catch (schedulingError) {
+      toast.push({
+        status: 'error',
+        title: (
+          <Text muted size={1}>
+            <Translate
+              t={t}
+              i18nKey="toast.schedule.error"
+              values={{title: release.metadata.title, error: schedulingError.message}}
+            />
+          </Text>
+        ),
+      })
+      console.error(schedulingError)
+    } finally {
+      setStatus('idle')
+    }
+  }, [
+    schedule,
+    release._id,
+    release.metadata.intendedPublishAt,
+    release.metadata.title,
+    telemetry,
+    toast,
+    t,
+  ])
+  const {t: coreT} = useTranslation()
```

Comment:
Not necessary before merging and a nitpick, but you probably don't need to rename this since you are only using one LocaleNamespace :)

Reply:
Ah nevermind just realised you are importing the namespace up there!